draft-ietf-quic-version-negotiation.md

```diff
@@ -143,8 +210,11 @@ the Version Negotiation packet, and a distinct connection after.
 
 ## Incompatible Version Negotiation {#incompat-vn}
 
-The server starts incompatible version negotiation by sending a VN packet,
-listing all the versions that it does support.
+The server starts incompatible version negotiation by sending a Version
+Negotiation packet. This packet SHALL list the server's set of Negotiated
+Versions (see {{server-fleet}}) in the packet's Supported Version field. The
```

Comment:
Nit:  Supported Version doesn't contain a list -- it contains a single version and repeats as many times as necessary to convey the list.

```suggestion
Negotiation packet. This packet SHALL include each entry from the server's
set of Negotiated Versions (see {{server-fleet}}) in a Supported Version
field. The
```

Reply:
Fixed.